src/components/Data/Collections/ModalClear.vue

```diff
@@ -71,9 +71,12 @@ export default {
       }
 
       try {
-        await this.$store.direct.dispatch.collection.clearCollection({
+        await this.$kuzzle.query({
```

Comment:
⚠️ this is not meant to work. Kuzzle SDK is now accessed via the store getter, which returns the version corresponding to the current environment major version. If `this.$kuzzle` works it means there's something weird going on.

Reply:
indeed it doesn't work, a small error of copy/paste